README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
The link provided displays a 404 error. Could there be a mistake in spelling or the wrong link attached?